src/file-upload/file-list-item/file-list-item-thumbnail.tsx

```diff
@@ -0,0 +1,73 @@
+import styled from "styled-components";
+import { Color } from "../../color";
+import { MediaQuery } from "../../media";
+import { TextStyleHelper } from "../../text";
+
+interface Props {
+    thumbnailImageDataUrl: string;
+    renderReplaceButton?: boolean | undefined;
+    onReplaceClick?: (() => void) | undefined;
```

Comment:
Likely to drop this feature in the near term. But will still dedicate some time to explore a better method in future. Can leave the code there for now